packages/extension/src/companion/Companion.tsx

```diff
@@ -113,6 +113,14 @@ export default function Companion({
     routeChangedCallbackRef.current();
   }, [containerRef]);
 
+  useEffect(() => {
```

Comment:
I think instead of doing `useEffect`, we should add `setCompanionState(true)` to  `onOpenComments`. Otherwise, it may feel like magic